auth-persistence/src/main/java/org/iqu/auth/persistence/dao/JdbcConstants.java

```diff
@@ -0,0 +1,15 @@
+package org.iqu.auth.persistence.dao;
+
+/**
+ * Use for database connection.
+ * 
+ * @author Mitroi Stefan-Daniel
+ *
+ */
+public class JdbcConstants {
+	public static final String DRIVER = "com.mysql.jdbc.Driver";
+	public static final String URL = "jdbc:mysql://localhost:3306/mydb";
```

Comment:
mydb? ar trebui un nume mai clar si username si parola cam urat sa le ai puse in fisier.